jerry-port/default/default-date.c

```diff
@@ -21,33 +24,32 @@
 #include "jerryscript-port-default.h"
 
 /**
- * Default implementation of jerry_port_get_time_zone. Uses 'gettimeofday' if
- * available on the system, does nothing otherwise.
+ * Default implementation of jerry_port_get_local_tza. Uses the 'tm_gmtoff' field
+ * of 'struct tm' (a GNU extension) filled by 'localtime_r' if available on the
+ * system, does nothing otherwise.
  *
- * @return true - if 'gettimeofday' is available and executed successfully,
- *         false - otherwise.
+ * @return offset between UTC and local time at the given unix timestamp, if
+ *         available. Otherwise, returns 0, assuming UTC time.
  */
-bool jerry_port_get_time_zone (jerry_time_zone_t *tz_p) /**< [out] time zone structure to fill */
+double jerry_port_get_local_tza (double js_time_ms,  /**< ms since unix epoch */
```

Comment:
I'm not sure about the `js_` prefix for this param. It's time in millisecs, which is quite well expressed by `time_ms`. There is not much JS-specific in that concept.

Reply:
I'd only done this because it was a `double`, but I agree, I think `unix_ms` is probably a better name. (or do you prefer `time_ms`? Both seem better than `js_time_ms`).

Reply:
`unix_ms` is fine by me